applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -1342,6 +1428,54 @@ static bool handle_hid_report_sent_event(
 	return false;
 }
 
+static void handle_keyboard_leds_report(struct subscriber *sub, uint8_t report_id,
+					const uint8_t *data, size_t len)
+{
+	BUILD_ASSERT(REPORT_SIZE_KEYBOARD_LEDS == 1);
+	BUILD_ASSERT(REPORT_SIZE_KEYBOARD_LEDS <= OUTPUT_REPORT_DATA_MAX_LEN);
+	__ASSERT_NO_MSG(report_id == REPORT_ID_KEYBOARD_LEDS);
```

Comment:
passing report_id as argument and this assert are not needed
this function is called when report_id is REPORT_ID_KEYBOARD_LEDS

Reply:
Changed